<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>762</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>210</y>
      <width>351</width>
      <height>30</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="usernamecombo">
     <item>
      <widget class="QLabel" name="username_label">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel
{	
	color: white;
}</string>
       </property>
       <property name="text">
        <string>Логин:  </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="input_login">
       <property name="font">
        <font>
         <family>Tw Cen MT</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit
{
background-color: rgba(255, 255, 254, 100);
border: none;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="going">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>370</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color: white;
border: 2px solid black;
border-radius: 10px; 
}</string>
    </property>
    <property name="text">
     <string>Войти</string>
    </property>
   </widget>
   <widget class="QLabel" name="titlerocket">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>40</y>
      <width>351</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>36</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{	
	
	color: rgb(255, 255, 127);
}</string>
    </property>
    <property name="text">
     <string>ВИКТОРИНА</string>
    </property>
   </widget>
   <widget class="QLabel" name="allowancelabel">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>140</y>
      <width>761</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{	
	color: white;
}</string>
    </property>
    <property name="text">
     <string>ВХОД</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="mistakes">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>310</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	
	color: rgb(255, 21, 44);
}</string>
    </property>
    <property name="text">
     <string>Неверный логин или пароль</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>260</y>
      <width>351</width>
      <height>30</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="password">
     <item>
      <widget class="QLabel" name="password_label">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel
{	
	color: white;
}</string>
       </property>
       <property name="text">
        <string>Пароль:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="input_password">
       <property name="font">
        <font>
         <family>Tw Cen MT</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit
{
background-color: rgba(255, 255, 254, 100);
border: none;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="rocket">
    <property name="geometry">
     <rect>
      <x>-90</x>
      <y>0</y>
      <width>851</width>
      <height>521</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Vladimir Script</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/помощник ракеты/hfrtnf yjvth 1 d hjccbbb.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>картинкни/psikhologiia-fon-12.webp</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="registrationbutton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>370</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color: white;
border: 2px solid black;
border-radius: 10px; 
}</string>
    </property>
    <property name="text">
     <string>Регистрация</string>
    </property>
   </widget>
   <zorder>rocket</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>going</zorder>
   <zorder>titlerocket</zorder>
   <zorder>allowancelabel</zorder>
   <zorder>mistakes</zorder>
   <zorder>layoutWidget_3</zorder>
   <zorder>registrationbutton</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>762</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="помощник банкомата.qrc"/>
  <include location="помощник ракеты2.qrc"/>
 </resources>
 <connections/>
</ui>
